lib/bundles/inspec-compliance/README.md

```diff
@@ -8,7 +8,8 @@ This extensions offers the following features:
 
 To use the CLI, this InSpec add-on adds the following commands:
 
- * `$ inspec compliance login user password` - authentication against Chef Compliance
+ * `$ inspec compliance api_token server --token TOKEN --user USER` - save the Chef Compliance API token for user
```

Comment:
I think this plugin should offer two methods
- handle the complete oauth flow to retrieve a token. that allows the plugin to get a refresh token as well (requires us to setup a specific client/application for inspec)
- take an api token as input